app/controllers/providers/application_merits_task/opponent_organisations_controller.rb

```diff
@@ -0,0 +1,62 @@
+module Providers
+  module ApplicationMeritsTask
+    class OpponentOrganisationsController < ProviderBaseController
+      def show
+        @form = Opponents::OrganisationForm.new(model: opponent, name: opponent.name, ccms_code: opponent.ccms_code, description: opponent.description)
+      end
+
+      def new
+        @organisation_types = organisation_types
+        @form = Opponents::OrganisationForm.new(model: opponent)
+      end
+
+    #   def update
+    #     @form = Opponents::OrganisationForm.new(form_params)
+    #     render :show unless update_task_save_continue_or_draft(:application, :opponent_name)
+    #   end
```

Comment:
I presume this method is required and that the comments should be removed.

Reply:
@naseberry you are looking at an early version of the code, this has changed in later commits.